diff --git a/classes/ttfontsuni.php b/classes/ttfontsuni.php
index b65a759..3f2bbde 100644
--- a/classes/ttfontsuni.php
+++ b/classes/ttfontsuni.php
@@ -610,7 +610,7 @@ class TTFontFile
 			throw new MpdfException("Could not find PostScript font name: " . $this->filename);
 		// CHECK IF psName valid (PadaukBook contains illegal characters in Name ID 6 i.e. Postscript Name)
 		$psNameInvalid = false;
-		for ($i = 0; $i < count($psName); $i++) {
+		for ($i = 0; $i < strlen($psName); $i++) {
 			$c = $psName[$i];
 			$oc = ord($c);
 			if ($oc > 126 || strpos(' [](){}<>/%', $c) !== false) {
